Upstatement/react-router-guards
# Switch from TravisCI to GitHub Actions

# Description

This moves the RRG test-runner off of TravisCI in favor of GitHub Actions

This is also in coordination with the major branch change to `main`

## What this does

- Removes `travis.yml`
- Adds new `.github/workflows/test.yml` file
- Updates `test` npm command to use watch mode, and `test:ci` to call the runner once (for CI environments)

## How to test

1. Confirm the test workflow runs on this PR
2. Confirm it runs successfully!
